public/app/plugins/datasource/prometheus/querybuilder/components/PromQueryBuilder.tsx

```diff
@@ -42,15 +43,94 @@ export const PromQueryBuilder = React.memo<Props>((props) => {
   const { datasource, query, onChange, onRunQuery, data, showExplain } = props;
   const [highlightedOp, setHighlightedOp] = useState<QueryBuilderOperation | undefined>();
   const [showDrawer, setShowDrawer] = useState<boolean>(false);
+  const [llmAppEnabled, updateLlmAppEnabled] = useState<boolean>(false);
 
   const lang = { grammar: promqlGrammar, name: 'promql' };
 
   const initHints = datasource.getInitHints();
 
+  useEffect(() => {
+    async function checkLlms() {
+      const check = await isLLMPluginEnabled();
+      updateLlmAppEnabled(check);
+    }
+    checkLlms();
+  }, []);
+
+  const theme = useTheme2();
+  const styles = getStyles(theme);
+
+  const queryAssistantButton = function () {
+    const llmAppDisabled = !llmAppEnabled;
+    const noMetricSelected = !query.metric;
+
+    const button = () => {
+      return (
+        <Button
+          variant={'secondary'}
+          onClick={() => {
+            reportInteraction('grafana_prometheus_promqail_ai_button_clicked', {
+              metric: query.metric,
+            });
+            setShowDrawer(true);
+          }}
+          disabled={!query.metric || !llmAppEnabled}
+        >
+          <img height={16} src={AI_Logo_color} alt="AI logo black and white" />
+          {'\u00A0'}Get query suggestions
```

Comment:
is the unicode nbsp/non breaking space here on purpose?

Reply:
Yep! I just wanted space between the image and the text.